### DISCUSSION TITLE
How can I get this event?

### DISCUSSION THREAD

**Discussion by novikov9999** [discord-php/DiscordPHP#1126] [Q&A] [answered]
Hi all. I'm testing an application and I need to receive a message from a bot in a separate channel. Regular messages are successfully received by the MESSAGE_CREATE event. How can I get such an event?

<img width="599" alt="Screenshot 2023-06-03 at 18 46 06" src="https://github.com/discord-php/DiscordPHP/assets/132385847/a741378d-6e90-446e-8f35-be5506351d5e">

My code does not react to the appearance of such an event.
```
include __DIR__.'/vendor/autoload.php';

use Discord\Discord;
use Discord\Parts\Channel\Message;
use Discord\WebSockets\Intents;
use Discord\WebSockets\Event;
use Discord\Parts\Interactions\Interaction;

$discord = new Discord([
    'token' => 'TOKEN',
	'intents' => Intents::getDefaultIntents() | Intents::GUILD_MEMBERS | Intents::GUILD_PRESENCES | Intents::GUILDS | Intents::GUILD_BANS, 
	'loadAllMembers' => true
]);



$discord->on('ready', function (Discord $discord) {

    $discord->on(Event::INTERACTION_CREATE, function (Message $interaction, Discord $discord) {
    	echo 'INTERACTION_CREATE'.PHP_EOL;
    });

});

$discord->run();
```

**Comment by key2peace** [comment, marked as answer]
it's an ephermal message (hence the text below it saying 'Only you can see this') which pretty much explains why your bot won't see it then

**Comment by novikov9999** [comment]
@key2peace Is there any how to see ephermal message? I had an idea to use web selenium, but for now I want to find a simpler solution.

I'm willing to pay for a consultation.
